homeassistant/components/binary_sensor/zwave.py

```diff
@@ -63,6 +63,11 @@ def should_poll(self):
         """No polling needed."""
         return False
 
+    @property
```

Comment:
The default is now to return None (use whole node). It is used in climate & lock.

I'll change the default to return [] and override in climate & lock to return None.